### DISCUSSION TITLE
Issue with CGAL ITK_Support

### DISCUSSION THREAD

**Discussion by oarcelus** [CGAL/cgal#7854] [Q&A] [answered]
I am trying to build a CGAL program that uses ITK.

I downloaded and setup everything with vcpkg for x64. When trying to configure my Cmake folder the following error appears.

CMake Error at C:/dev/vcpkg/installed/x64-windows/share/cgal/CGAL_ITK_support.cmake:4 (set_target_properties):
  Property INTERFACE_LINK_LIBRARIES may not contain link-type keyword
  "optimized".  The INTERFACE_LINK_LIBRARIES property may contain
  configuration-sensitive generator-expressions which may be used to specify
  per-configuration rules.
Call Stack (most recent call first):
  CMakeLists.txt:52 (include)

Does this mean that I need to install ITK in ´debug´ or ´general´ mode?

**Comment by lrineau** [comment, marked as answer]
That is a bug from about file `CGAL_ITK_support.cmake`. That file should use the imported targets created by `ITKConfig.cmake`.

I have created the issue https://github.com/CGAL/cgal/issues/7855, from this discussion.

**Comment by janetournois** [comment]
PR #7858 should fix this issue
